publish_map:  false # Whether to publish a grid map, either created from the coordinates or received from a map server.
cell_warn:    1000  # Number of grid cells in the map above which a performance warning is issued.
wait_for_map: 2.0   # Time in seconds to wait initially for another map provider.
